hack/lib/util.sh

```diff
@@ -720,8 +720,24 @@ function kube::util::ensure-cfssl {
     cfssldir="${KUBE_TEMP}/cfssl"
   fi
 
-  mkdir -p "${cfssldir}"
-  pushd "${cfssldir}" > /dev/null
+  case "$(uname -m)" in
```

Comment:
We'd better not add another exact same logic method (the same with that in #72772) in hack scripts.

I've created a PR to add a consistent environment. Your PR can rely on this.

Reply:
Done.